src/cdfmean.f90

```diff
@@ -115,10 +124,15 @@ PROGRAM cdfmean
      PRINT *,'                   partial steps.'
      PRINT *,'       [ -var ]  : also compute the spatial variance of cdfvar '
      PRINT *,'       [ -zeromean ] : create a file with cdfvar having a zero spatial mean.'
+     PRINT *,'       [ -out_suff OUT-file-suffix ] : specify SUFFIX (+ directory) of the output files'
+     PRINT *,'       [ -basins n_basins basin_names.. ] : specify basins in ',TRIM(cn_fbasins),' to be used'
```

Comment:
In the commit 7692fcd of modcdfname, I move the subasin variable name and file name to the NEMO one (like this no need to rename this file and the variable). To compute quantities over a specific domain (lets say Caspian sea for example) you can build a 3d mask of the Caspian sea with cdfmkmask and run your tool with this file. I agree for eORCA12, the most convenient could be to multiply the 2d mask instead of the 3d mask. We have to be clever enough to not kill the default behaviour (ie if subbasin there, use the default variable name).